web/src/components/layouts/routes.tsx

```diff
@@ -159,6 +161,13 @@ export const ROUTES: Route[] = [
     newTab: true,
     customizableHref: "documentationHref",
   },
+  {
+    title: "Background Migrations",
+    pathname: "/background-migrations",
+    icon: ArrowUp10,
+    bottom: true,
+    show: () => !Boolean(env.NEXT_PUBLIC_LANGFUSE_CLOUD_REGION),
```

Comment:
IMO this might be a better indicator than any cloud plan. This should never be set for self-hosted environments.